lib/performance/use_case/sync_s3_to_elasticsearch.rb

```diff
@@ -10,8 +10,11 @@ def initialize(s3_gateway:, elasticsearch_gateway:, logger: Logger.new(STDOUT))
   def execute
     record_count = 0
 
-    @s3_gateway.each do |_key, data|
-      @elasticsearch_gateway.write(data)
+    @s3_gateway.each do |key, data|
```

Comment:
maybe we could write a script that adds the 'date' field to all existing records in S3 then we don't  need to do that here. We also need to add a 'date' field to all the `PerformancePlatform::Gateway::<metric>` classes so the right data is uploaded to ES and S3 daily and if we do that then 'new' values in S3 will have a date field and the older ones don't. What do you think?